nixos/modules/services/misc/bazel-remote.nix

```diff
@@ -0,0 +1,146 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.bazelRemote;
+  condOption = name: value: if value != null then " --${name} ${toString value}" else "";
+  bazelRemoteConfig
+    = condOption "config_file" cfg.yamlConfig
+    + condOption "dir" cfg.dir
+    + condOption "max_size" cfg.maxSize
+    + condOption "host" cfg.bind
+    + condOption "port" cfg.port
+    + condOption "htpasswd_file" cfg.htpasswdFile
+    + condOption "tls_cert_file" cfg.tlsCertFile
+    + condOption "tls_key_file" cfg.tlsKeyFile
+    + condOption "idle_timeout" cfg.idleTimeout
+    ;
+in
+{
+
+  ###### interface
+
+  options = {
+
+    services.bazelRemote = {
+
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Whether to enable the Bazel Remote server.";
+      };
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.bazel-remote;
+        defaultText = "pkgs.bazelRemote";
```

Comment:
That's not right

Reply:
What should this be? It was a copy and paste from another module (can't remember which one)

Reply:
pkgs.bazelRemote doesn't exist